GitHub action fails with a 500 response

We're suddenly seeing the `glean-probe-scraper` GitHub action in [the bedrock repo](https://github.com/mozilla/bedrock/) fail with a 500 error `Your "GCE" credentials are invalid`.

https://github.com/mozilla/bedrock/actions/runs/3573138621/jobs/6006893479

```
Run curl --fail-with-body https://us-west1-moz-fx-data-probe-s-prod-2bc3.cloudfunctions.net/glean-push --data '{
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   130    0     0  100   130      0    106  0:00:01  0:00:01 --:--:--   106
100   130    0     0  100   130      0     58  0:00:02  0:00:02 --:--:--    58
100   130    0     0  100   130      0     40  0:00:03  0:00:03 --:--:--    40
100   130    0     0  100   130      0     30  0:00:04  0:00:04 --:--:--    30
100   130    0     0  100   130      0     24  0:00:05  0:00:05 --:--:--    24
100   670  100   540  100   130     97     23  0:00:05  0:00:05 --:--:--   125
100   670  100   540  100   130     97     23  0:00:05  0:00:05 --:--:--   163
curl: (22) The requested URL returned error: 500
Error: Command ['gsutil', '-q', '-m', 'rsync', '-r', 'gs://probe-scraper-prod-artifacts/glean/bedrock/', '/tmp/tmpishnj5is/output/glean/bedrock'] returned non-zero exit status 1: Your "GCE" credentials are invalid. For more help, see "gsutil help creds", or re-run the gsutil config command (see "gsutil help config").
Caught non-retryable exception while listing gs://probe-scraper-prod-artifacts/glean/bedrock/: [Errno 30] Read-only file system: '/www-data-home/.gsutil'
CommandException: Caught non-retryable exception - aborting rsync

Error: Process completed with exit code 22.
```

--- a/.gce_boto
+++ b/.gce_boto
@@ -1,2 +1,5 @@
 [GoogleCompute]
 service_account = default
+
+[GSUtil]
+state_dir = /tmp/gsutil

